README.md

```diff
@@ -181,50 +232,45 @@ message BookInfo {
    string title = 2;
    string author = 3;
 }
-
-message MyLibrary {
-   int64 id = 1;
-   string name = 2;
-   repeated BookInfo books = 3;
-   map<string,string> keys = 4;
-}
 ```
 
-After saving the above, you can generate Swift code using the following command:
-
+Then generate Swift code using:
 ```
 $ protoc --swift_out=. DataModel.proto
 ```
 
-This will create a file `DataModel.pb.swift` with a `struct BookInfo` and a
-`struct MyLibrary` with corresponding Swift fields for each of the proto fields
-and a host of other capabilities:
+The generated code will expose a Swift property for
+each of the proto fields as well as a selection
+of serialization and deserialization capabilities:
+```swift
+// Create a BookInfo object and populate it:
+var info = BookInfo()
+info.id = 1734
+info.title = "Really Interesting Book"
+info.author = "Jane Smith"
+
+// As above, but generating a read-only value:
+let info2 = BookInfo. with {
+    $0.id = 1735
+    $0.title = "Even More Interesting"
+    $0.author = "Jane Q. Smith"
+  }
+
+// Serialize to binary protobuf format:
+let binaryData: Data = info.serializedData()
```

Comment:
I believe most of these need `try`, up to you if you want to also include examples of error handling.